.github/workflows/test-run-acc-tests-for-path.yml

```diff
@@ -22,7 +22,7 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - id: get-metadata
-        run: echo "go-version=$(cat ./.go-version)" >> $GITHUB_OUTPUT
+        run: echo "go-version=$(cat ./.go-version)" >> "$GITHUB_OUTPUT"
       - name: Set Up Go
         uses: actions/setup-go@v3
         with:
```

Comment:
:thinking: We should be able to get rid of the `get-metadata` step here, see comment on `.github/workflows/godoc-test-checker.yml` 😄